contact.csv (3).ipynb

```diff
@@ -0,0 +1,173 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25264925",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9b312719",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file=open('contact.csv','w',newline='')"
```

Comment:
open file with another way

Reply:
we can use here "with open" syntatx

Reply:
with open('filename.csv','w',newline='') as f:

Reply:
yes

Reply:
Output ?